example/lib/src/app/pages/home/home_view.dart

```diff
@@ -31,41 +31,60 @@ class _HomePageState extends ViewState<HomePage, HomeController> {
           child: Column(
             mainAxisAlignment: MainAxisAlignment.center,
             children: <Widget>[
-              Text(
-                // use data provided by the controller
-                'Button pressed ${controller.counter} times.',
+              ControlledWidget<HomeController>(
+                builder: (context, controller) {
```

Comment:
Can this be wrapped more cleanly in such a way that the user does not have to write this every time?

Reply:
I choosed this approach (like Material's `Builder`) so the dev can have control of context and controller in the time of the definition.

The code will be a lot less confuse when you apply it correctly

Example:

```dart
class TitleWidget extends StatelessWidget {
	Widget build(BuildContext context) {
		return ControlledWidgetBuilder<HomeController>(
	        builder: (context, controller) {
	          return Text(controller.title);
	        },
	      );
	}
}
```

And then on HomePage, just use `TitleWidget()` and the rest as usual. 

At the end, it will be a similar approach when you call `FlutterCleanArchitecture.getController` on a `StatefulWidget`, but the idea is that the widgets will be created separately and organized like that, so the code keeps organized.

Reply:
@ShadyBoukhary But to keep clarity, I renamed it to `ControlledWidgetBuilder`, to keeps cohesion with class proposal.